homeassistant/components/systemmonitor/sensor.py

```diff
@@ -108,7 +108,7 @@
     "processor_temperature": (
         "Processor temperature",
         TEMP_CELSIUS,
-        CPU_ICON,
```

Comment:
We should set a device class instead.

Reply:
and I thought this was an easy fix..
do you have an example integration using that for a comparable sensor please?

Reply:
import `DEVICE_CLASS_TEMPERATURE` from `homeassistant.const` (top of the file, more of those imports).

Change: 
- `"mdi:thermometer"` -> `None`
- The next line says `None` -> `DEVICE_CLASS_TEMPERATURE`

That should do the trick

Reply:
thanks, managed that I think ;-)

Reply:
btw while we're at it, shouldn't https://github.com/home-assistant/core/blob/39a976c6fe4e43f1d1cc789ba9fdf5ca5c3f11b6/homeassistant/components/systemmonitor/sensor.py#L76 use None for icon too then, give the use of the device_class there?

Reply:
Yep, we have some of these cases still in our code base, but yeah, in this case, it doesn't need an icon set.

Reply:
ok, let me take it out in one go now.